### DISCUSSION TITLE
🙏 is there any guide, to integrate Node.js with OAM ?

### DISCUSSION THREAD

**Discussion by bharatsahulw** [bergie/passport-saml#834] [Q&A]
Hi All,

Is there any way to integrate the node js with passport-saml by using the OAM metadata file instead of ADFS metadata file? I am using below keys from the metadata file in the passport-saml. 

passport.use(/login, new SamlStrategy({
            path: "login_file_path",
            entryPoint: "**Entry url from the metadata file or IDP/SP entry url**",
            issuer: "my application url",
            cert: "**<x509:certificate> from the metadata file**",
            disableRequestedAuthnContext: true,
            authnContext: 'http://schemas.microsoft.com/ws/2008/06/identity/authenticationmethod/windows'
        }, function(user, done) {
            if (user && user.nameID) {
               done (null, user)
           } else {
                done(null, null);
            }
        }));

**Comment by srd90** [comment]
for the record: @bharatsahulw created later on issue #835 with similar content than this discussion ( #834 ) so anyone reading this discussion should look extra info from #835
